packages/suite/src/views/wallet/coinmarket/buy/offers/Offers/List/Quote/index.tsx

```diff
@@ -212,12 +215,11 @@ const Quote = ({ className, quote, wantCrypto }: Props) => {
     const { selectQuote, providersInfo } = useCoinmarketBuyOffersContext();
     // TODO - tags are not yet fully supported by the API server
```

Comment:
I would remove the TODO from here - we are handling the visual tag at this moment and when/if we extend the `getTagAndInfoNote`, we will know what to do.

Reply:
Removed.